WordPress/Classes/ViewRelated/NUX/LoginViewModel.m

```diff
@@ -187,6 +187,8 @@ - (BOOL)isOnePasswordEnabled
 
 - (void)requestOneTimeCode
 {
+    NSString *message = NSLocalizedString(@"Sending SMS!", @"The app will request One Time Code to be sent via SMS");
```

Comment:
Now that you mention it, Notifications > Reply works the same way. It says 'Sent' right away.

I'll update the string, thanks for the feedback @koke!